service/history/replicatorQueueProcessor.go

```diff
@@ -717,7 +717,7 @@ func (p *replicatorQueueProcessorImpl) getVersionHistoryItems(
 		return nil, nil, serviceerror.NewInternal("replicatorQueueProcessor encounter workflow without version histories")
 	}
 
-	versionHistoryIndex, err := versionhistory.FindFirstVersionHistoryIndexByItem(
+	versionHistoryIndex, err := versionhistory.FindItemFirstIndex(
```

Comment:
i would prefer keeping the current function names

Reply:
In this case `Item` is clearly `VersionHistoryItem`.

Reply:
FindItemFirstIndex means different.

`FindFirstVersionHistoryIndexByItem` this function tries to find the first version history which contains the item